src/solve.jl

```diff
@@ -749,10 +752,17 @@ function many_solve(
     threading::Bool,
     catch_interrupt::Bool,
 ) where {flatten}
+
+
+    if isa(starts, TotalDegreeStartSolutionsIterator)
+        @warn "Solving for many parameters with total degree start system is not recommended. Instead, one should use a two-step approach: first solve a system with generic parameters, and track its solutions to the desired parameters. See https://www.juliahomotopycontinuation.org/guides/many-systems/."
+    elseif isa(starts, PolyhedralStartSolutionsIterator)
+        @error "Solving for many parameters with polyhedral start system is not implemented and also not recommended. Instead, one should use a two-step approach: solve a system with generic parameters, and track its solutions to the desired parameters. See https://www.juliahomotopycontinuation.org/guides/many-systems/."
+    end
```

Comment:
What is stopping us from generating random start solutions with generic parameters in the if statements and passing this to `threaded_solve(solver, starts; catch_interrupt = false)`?

Reply:
> What is stopping us from generating random start solutions with generic parameters in the if statements and passing this to `threaded_solve(solver, starts; catch_interrupt = false)`?

Because we would have to also update the structure of `solver`. This would require me to go deep into the code of  `solve` and I prefer to keep this simple solution.

Reply:
Okay :)